Library/Homebrew/dependencies.rb

```diff
@@ -39,11 +39,6 @@ def dup_without_system_deps
   def inspect
     "#<#{self.class.name}: #{__getobj__}>"
   end
-
-  sig { returns(T::Array[Dependency]) }
-  def to_a
```

Comment:
I fixed the corresponding rbi file to support all the enumerable methods of the delegated class, which also means we don't need this explicit forwarding `def`.